web/src/app/browser/src/contextManager.ts

```diff
@@ -211,6 +211,12 @@ export default class ContextManager extends ContextManagerBase<BrowserConfigurat
     const originalTarget = this.activeTarget; // may differ, depending on focus state.
 
     if(target == originalTarget) {
+      // A focus state may have .currentTarget as null at this stage; if the func
+      // is being called with a non-null parameter, we want this SET.
+      if(target) {
```

Comment:
```suggestion
      if(originalTarget) {
```

It comes to the same thing but is less cognitive load to read I think!